src/Microsoft.Health.Fhir.R4.Web/appsettings.json

```diff
@@ -38,7 +38,10 @@
         },
         "Operations": {
             "Export": {
-                "Enabled": false,
+                "Enabled": true,
+                "StorageAccountType": null,
```

Comment:
Can this be move to top level? E.g., a storage account that can be used for other purpose too not just export?

Reply:
I feel that it would be okay to keep it within export for now and move it later when we decide to use it for other purposes.